morpheus/pipeline/receiver.py

```diff
@@ -71,58 +72,122 @@ def is_partial(self):
 
     @property
     def in_pair(self):
-        return (self.in_stream, self.in_pair)
+        return (self.in_node, self.in_pair)
 
     @property
-    def in_stream(self):
-        return self._input_stream
+    def in_node(self):
+        return self._input_node
 
     @property
     def in_type(self):
         return self._input_type
 
     def get_input_pair(self, builder: mrc.Builder) -> StreamPair:
         """
+        TODO: Remove this function
```

Comment:
Remark: Looks like this whole function is going to get deleted, so I'll skip review of it.